### ISSUE TITLE
Workflow: publish histories matching a title and grab links

### ISSUE THREAD

**Issue by hexylena** [galaxy-iuc/parsec#8]
I almost wrote a tool for this since I've done this for 20+ histories x 3 times this semester. Clearly should be JQ + parsec.

**Comment by hexylena** [comment]
```
parsec histories_get_histories | \
    jq '.[] | select(.name | contains("2017.04.27")) | .id'  -r | \
    xargs parsec histories_update_history --importable true
```

- needs new bioblend release with content_type bug fixed
- needs updated CLI tools
- needs new args in CLI

**Comment by hexylena** [comment]
Fixed in #10
